lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -1682,6 +1682,11 @@ definitions:
         description: VLAN ID for management interface traffic on the AGW
         minLength: 1
         maxLength: 4
+      sgi_management_iface_static_ip:
+        type: string
```

Comment:
There is `format: ipv4` for IP addresses in swagger. Grep in the codebase for existing usages